src/error.rs

```diff
@@ -41,15 +41,12 @@ pub enum ErrorKind {
         "Unexpected error: {0}. Please report this error on https://github.com/mimoo/noname/issues"
     )]
     UnexpectedError(&'static str),
```

Comment:
maybe a good idea to change the comment there, since we don't use it for testing anymore, but to warn users with nicer errors than panics